docs/sources/dashboards/build-dashboards/manage-library-panels/index.md

```diff
@@ -24,6 +24,8 @@ A library panel is a reusable panel that you can use in any dashboard. When you
 
 You can save a library panel in a folder alongside saved dashboards.
 
+**Note:** When [RBAC](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/) is enabled, you need to have the relevant [Permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/) to create, manage, and view library panels.
```

Comment:
```suggestion
{{% admonition type="note" %}}
When [RBAC](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/) is enabled, you need to have the relevant [Permissions](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/) to create, manage, and view library panels.
{{% /admonition %}}
```

Reply:
FYI: I'm using version interpolation syntax in the links which ensures that it always points to the right place.